.github/dependabot.yml

```diff
@@ -22,13 +22,16 @@ updates:
       - dependency-name: "envoyproxy/toolshed/gh-actions/retest"
     groups:
       github-actions:
-        - "actions*"
+        patterns:
+          - "actions*"
   - package-ecosystem: gomod
     directories:
       - "/"
-      - "examples/extension-server"
     schedule:
       interval: weekly
+    ignore:
+      # skip to update cel-go, because it related to k8s.io/apiserver
```

Comment:
can we link any GH issue / link here, as a hint to figure out when to rm this section ?

Reply:
we should let tie cel-go to k8s.io/apiserver.